x-pack/platform/plugins/private/intercepts/kibana.jsonc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "plugin",
+  "id": "@kbn/intercepts-plugin",
+  "owner": ["@elastic/appex-sharedux"],
+  "group": "platform",
+  "visibility": "private",
```

Comment:
Are you sure you don't want other plugins to use any apis the `intercepts` plugin exposes? If other plugins are supposed to depend on the `intercepts`, it should probably be set to `visibility: shared`

Reply:
I actually want other plugins to depend on this, I'll change the visibility here.